les/vflux/server/prioritypool.go

```diff
@@ -320,36 +324,45 @@ func (pp *priorityPool) disconnectedNode(c *ppNodeInfo) {
 	pp.activeQueue.Remove(c.activeIndex)
 	pp.inactiveQueue.Remove(c.inactiveIndex)
 	if c.capacity != 0 {
-		pp.setCapacity(c, 0)
-		updates = pp.tryActivate()
+		pp.setTempState(c)
+		pp.setTempCapacity(c, 0)
+		updates = pp.tryActivate(true)
 	}
 }
 
-// markForChange internally puts a node in a temporary state that can either be reverted
+// setTempState internally puts a node in a temporary state that can either be reverted
 // or confirmed later. This temporary state allows changing the capacity of a node and
 // moving it between the active and inactive queue. activeFlag/inactiveFlag and
 // capacityField are not changed while the changes are still temporary.
-func (pp *priorityPool) markForChange(c *ppNodeInfo) {
-	if c.changed {
+func (pp *priorityPool) setTempState(c *ppNodeInfo) {
+	if c.tempState {
 		return
 	}
-	c.changed = true
-	c.origCap = c.capacity
-	pp.changed = append(pp.changed, c)
+	c.tempState = true
+	if c.tempCapacity != c.capacity { // should never happen
+		log.Crit("tempCapacity != capacity when entering tempState")
```

Comment:
Can we lower this log level to Error? I am afraid that the LES part has some issues and will just shutdown the entire Geth node.